auth-server/src/services/auth.ts

```diff
@@ -149,7 +150,7 @@ export default class AuthService {
 		const decoded = jwt.verify(token, publicKey)
 
 		if (isNaN(decoded.sub)) {
-			throw new Error('Invalid user id')
+			throw new ForbiddenError('Invalid user id in token')
```

Comment:
Shouldn't this rather be a `AuthenticationError` because it comes from the token?